spec/models/ont/run_factory_spec.rb

```diff
@@ -133,7 +239,11 @@
 
           it 'does not validate any flowcells' do
             allow(Ont::Flowcell).to receive(:new).and_return(flowcell)
-            expect(flowcell).to_not receive(:valid?)
+            # TODO: Ideally we'd not validate any flow cells,
```

Comment:
Why don't we need to validate (the new) flowcells explicitly? Not doing so allows you to submit a library_name that does not exist. Does validating the run validate the flowcells by association?

Reply:
Yes it does ... this is proven by the unit tests you added around validation.

Reply:
Should checking the run validates flowcells be in the run_spec tests instead of this run factory?

I'm not following why this test should expect flowcells to be validated when we choose not to validate the run, and don't explicitly validate any flowcells in the run factory